src/OpenTelemetry.Extensions/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
 * Removes .NET Framework 4.6.1. The minimum .NET Framework version
   supported is .NET 4.6.2.
 
+  * Removes net5.0 target as .NET 5.0 is going out
+  of support. The package keeps netstandard2.0 target, so it
+  can still be used with .NET5.0 apps.
```

Comment:
This is not without its risks - though perhaps minor 🤞.

Here's an example of a potential consequence https://github.com/open-telemetry/opentelemetry-dotnet/pull/3222#discussion_r921392854. We removed `net5.0` target from the Zipkin exporter and now .NET 5 apps loose out on an optimization.